include/umf/providers/provider_level_zero.h

```diff
@@ -83,6 +83,14 @@ umf_result_t umfLevelZeroMemoryProviderParamsSetFreePolicy(
     umf_level_zero_memory_provider_params_handle_t hParams,
     umf_level_zero_memory_provider_free_policy_t policy);
 
+/// @brief  Set the device ordinal in the parameters struct.
+/// @param  hParams handle to the parameters of the Level Zero Memory Provider.
+/// @param  deviceOrdinal device ordinal.
+/// @return UMF_RESULT_SUCCESS on success or appropriate error code on failure.
+umf_result_t umfLevelZeroMemoryProviderSetDeviceOrdinal(
```

Comment:
please add this function to def/map files

Reply:
Done.